app/src/test/java/org/mozilla/focus/shortcut/IconGeneratorTest.java

```diff
@@ -14,47 +14,47 @@
 public class IconGeneratorTest {
     @Test
     public void testRepresentativeCharacter() {
-        assertEquals("M", IconGenerator.getRepresentativeCharacter("https://mozilla.org"));
-        assertEquals("W", IconGenerator.getRepresentativeCharacter("http://wikipedia.org"));
-        assertEquals("P", IconGenerator.getRepresentativeCharacter("http://plus.google.com"));
-        assertEquals("E", IconGenerator.getRepresentativeCharacter("https://en.m.wikipedia.org/wiki/Main_Page"));
+        assertEquals("M", IconGenerator.Companion.getRepresentativeCharacter("https://mozilla.org"));
+        assertEquals("W", IconGenerator.Companion.getRepresentativeCharacter("http://wikipedia.org"));
+        assertEquals("P", IconGenerator.Companion.getRepresentativeCharacter("http://plus.google.com"));
+        assertEquals("E", IconGenerator.Companion.getRepresentativeCharacter("https://en.m.wikipedia.org/wiki/Main_Page"));
```

Comment:
Should we convert the test too? Or use @JvmStatic here too? But I guess converting the test is pretty straight-forward. :)

Reply:
Oh, just saw that this happens in the next commit. :)